magento/pwa-studio
# [chore] Update eslint dep

## Description

This is a follow up to the `magento-eslint` security issue. I updated our usage to `v1.5.0` and in doing so had to handle a bunch of new warnings/errors. I also had to update some peer dependencies and install the config as a dependency to buildpack and peregrine where it was not before.

Question: Should this be a patch? People using it with custom components may start getting warnings for things.

## Related Issue





Closes [PWA-193](https://jira.corp.magento.com/browse/PWA-193).

## Acceptance 



### Verification Stakeholders
@jimbo @zetlen 

### Specification



### Verification Steps


1. `yarn && yarn lint`, no errors.
2. Make sure all the disables make sense. If they don't let's fix it here.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
